lms/templates/discussion/_thread_list_template.html

```diff
@@ -1,17 +1,6 @@
 <%page expression_filter="h"/>
 <%! from django.utils.translation import ugettext as _ %>
 <script type="text/template" id="thread-list-template">
-    <div class="forum-nav-header">
-        <button type="button" class="forum-nav-browse" id="forum-nav-browse" aria-haspopup="true">
```

Comment:
I don't see "All Topics" as being a menu, despite us adding the hamburger menu icon to it. It doesn't pop anything up, but instead acts just like a link.